deployment/terraform/create.go

```diff
@@ -160,22 +174,69 @@ func (t *Terraform) Create() error {
 
 			// Starting mattermost.
 			mlog.Info("Starting mattermost", mlog.String("host", ip))
-			cmd := fmt.Sprintf("sudo service mattermost start")
+			cmd := "sudo service mattermost start"
 			if err := sshc.RunCommand(cmd); err != nil {
 				mlog.Error("error running ssh command", mlog.String("cmd", cmd), mlog.Err(err))
 				return
 			}
 		}()
 	}
+}
 
-	// TODO: display the entire cluster info from terraformOutput later
-	// when we have cluster support.
-	mlog.Info("Deployment complete.")
+func (t *Terraform) setupProxyServer(output *terraformOutput, extAgent *ssh.ExtAgent) {
+	if len(output.ProxyIP.Value) == 0 {
```

Comment:
It's just a single server now. So `ProxyIP.Value` should give you the IP after we make the required changes in output.tf.